acquire/acquire.py

```diff
@@ -122,7 +122,13 @@ def misc_windows_user_homes(target):
                 yield misc_path
 
 
-def from_user_home(target, path):
+def from_user_home(target: Target, path: str):
+    if target.os == "osx":
+        # Until getting the users from osx is implemented in dissect.target
+        for homedir in target.fs.path("/Users/").glob("*"):
+            yield str(homedir.joinpath(path))
+        # Return after yielding all osx users in the user directory.
+        return
```

Comment:
Would it be an idea to split this off into a `misc_osx_user_homes()`, similar as for windows? I think we also want to iterate over `/Users/*` (and `/home/*` and `/root` for other Unixes), same as we do for windows, irrespective of what comes back from `user_details()`